pyscripts/horton_set_image_params.py

```diff
@@ -40,7 +61,17 @@ def get_deployment_model_json(self, json_filename):
             with open(json_filename, 'r') as f:
                 json_manifest = json.loads(f.read())
         except:
-            print(Fore.RED + "ERROR: in JSON manifest: " + json_filename, file=sys.stderr)
+            print(Fore.RED + "ERROR: in JSON manifest: " + json_filename + Fore.RESET, file=sys.stderr)
             traceback.print_exc()
-            print(Fore.RESET, file=sys.stderr)
+            sys.exit(-1)
         return json_manifest
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Update ImageParams Horton Manifest')
+    parser.add_argument('--manifest', required=True, help='path and filename of manifest', type=str)
+    parser.add_argument('--object_name', required=True, help='objectName in manifest', type=str)
+    parser.add_argument('--image_path', default='', help='imagePath in manifest to update', type=str)
+    parser.add_argument('--image_tag', default='', help='imageTag in manifest to update', type=str)
+    parser.add_argument('--create_options', default='', help='createOptions in manifest to update (dict)', type=str)
+    arguments = parser.parse_args(sys.argv[1:])
```

Comment:
>sys.argv[1:] [](start = 34, length = 12)

What's the difference between parse_args() with no parameters and parse_args(sys.argv[1:]).  I can see that you're pulling off the first arg (the program name) but why don't you accept the default behavior?